src/Mod/Sketcher/Gui/TaskSketcherMessages.cpp

```diff
@@ -41,30 +41,18 @@ using namespace SketcherGui;
 using namespace Gui::TaskView;
 namespace bp = boost::placeholders;
 
-TaskSketcherMessages::TaskSketcherMessages(ViewProviderSketch *sketchView) :
-    TaskBox(Gui::BitmapFactory().pixmap("document-new"), tr("Solver messages"), true, nullptr),
+SketcherSolverWidget::SketcherSolverWidget(ViewProviderSketch *sketchView, QWidget* parent) :

```

Comment:
Ok no renaming then.
I put the parent argument because all other taskbox widgets have it (example: SketcherGeneralWidget). And they are not used either. So I thought there is a reason for it